### DISCUSSION TITLE
DPM++ 2M SDE Karras sampler is missing from samplers list.

### DISCUSSION THREAD

**Discussion by cloudaxes** [AbdBarho/stable-diffusion-webui-docker#536] [Ideas]
A new sampler, DPM++ 2M SDE Karras, was added to the https://github.com/crowsonkb/k-diffusion.git repository in May.  This repository needs to be updated to use the sampler.

**Comment by cloudaxes** [comment]
Addressed in this pull request: https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/537

**Comment by AbdBarho** [comment]
PR merged!
